src/sgame/sg_admin.cpp

```diff
@@ -1702,9 +1705,17 @@ static void G_admin_ban_message(
 
 	if ( areason && ent )
 	{
+		trap_GetUserinfo( ent->num(), userinfo, sizeof( userinfo ) );
+
+		const char *s = Info_ValueForKey( userinfo, "name" );
+
+		G_ClientCleanName( s, name, sizeof( name ), ent->client );
```

Comment:
I think you should put nullptr for client here because if not it may do some goofy thing with incrementing a counter of the number of unnamed players...

This sort of thing is why we need refactoring :)